requestid/requestid.go

```diff
@@ -0,0 +1,36 @@
+package requestid
+
+import (
+	"context"
+	"github.com/google/uuid"
+	"github.com/infobloxopen/atlas-app-toolkit/gateway"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/metadata"
+)
+
+// DefaultRequestIDKey is the metadata key name for request ID
+var DefaultRequestIDKey = "Request-Id"
```

Comment:
How can I override `default` key? I see only one  way is to assign new value to the global public variable. Is it correct?